Makefile

```
@@ -144,26 +144,32 @@ clean:
-	$(Q) rm -rf ${RPMOUT_DIR}
+	$(Q) rm -rf $(RPMOUT_DIR)
 	$(Q) rpmbuild -bb --clean --define="version ${VERSION}" --define="_builddir `pwd`" --define="dist $(DISCOVERY_CLIENT_RELEASE).$*" --define "_rpmdir $(RPMOUT_DIR)" discovery-client.spec
 
-discovery-client-debs: VERSION = $(or $(LIGHTOS_VERSION),$(DEFAULT_REL))
-discovery-client-debs:
-	$(Q) rpmbuild -bb --clean --define="version ${VERSION}" --define="_builddir `pwd`" --define="dist $(DISCOVERY_CLIENT_RELEASE)" --define "_rpmdir $(RPMOUT_DIR)" discovery-client.spec
-	(cd build/dist && sudo alien --to-deb -v -k ${RPMOUT_DIR}/x86_64/discovery-client-${VERSION}-${DISCOVERY_CLIENT_RELEASE}.x86_64.rpm && sudo chown ${USER}:${USER} ${WORKSPACE_TOP}/discovery-client/build/dist/*.deb)
+# Per-distro work dir so concurrent el8/el9 deb builds don't share a path.
+discovery-client-debs-%: VERSION = $(or $(LIGHTOS_VERSION),$(DEFAULT_REL))
+discovery-client-debs-%: DEB_WORK = $(WORKSPACE_TOP)/discovery-client/build/deb/$*
+discovery-client-debs-%:
```

checkmake (0.3.2)

[warning] Target body for "discovery-client-debs-%" exceeds allowed length of 5 lines (6).

(maxbodylength)